public/app/plugins/datasource/graphite/specs/datasource.test.ts

```diff
@@ -2,15 +2,15 @@ import { GraphiteDatasource } from '../datasource';
 import _ from 'lodash';
 // @ts-ignore
 import $q from 'q';
-import { TemplateSrvStub } from 'test/specs/helpers';
```

Comment:
At first, I tried to use the `TemplateSrvStub` to test my change (shoehorning logic to mimic what the real one does)  - after several futile attempts, I opted out of it. At this point, the stub is completely different from the actual implementation and any attempts to try and backfill enter the land of diminishing returns. 

I noticed there were no hard dependencies on the actual `TemplateSrv` so I went ahead and used that with great success. There might be a case to remove the stub entirely from all the suite (it's still used in a couple of other places) but I did not explore that.